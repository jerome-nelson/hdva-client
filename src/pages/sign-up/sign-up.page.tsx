import React from "react";
import { HeaderTitle } from "components/header/header";
import { TextField, IconButton, Fab } from "@material-ui/core";
import { ArrowForward } from "@material-ui/icons";

export const SignUpPage = () => (
    <React.Fragment>
        <HeaderTitle title="Email Please" />
        <p>This is used to send you updates when a property is added and to reset your password if needed</p>
        <form>
                <TextField fullWidth id="standard-basic" label="Enter email" />
                <div>
                <IconButton aria-label="Submit Form" >
                    <Fab color="primary">
                        <ArrowForward />
                    </Fab>
                </IconButton>
                </div>
            </form>
    </React.Fragment>
);