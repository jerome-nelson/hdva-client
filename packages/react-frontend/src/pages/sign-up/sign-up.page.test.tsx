import React from "react";

import { render } from '@testing-library/react';
import { SignUpPage } from "./sign-up.page";

describe("Sign Up Page", () => {
    it("should match snapshot", () => {
        const result = render(
                <SignUpPage />
        );
        expect(result).toMatchSnapshot();
    });
});