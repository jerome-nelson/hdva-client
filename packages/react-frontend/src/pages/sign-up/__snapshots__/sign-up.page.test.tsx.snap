// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Sign Up Page should match snapshot 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <div
        class="MuiGrid-root MuiGrid-container MuiGrid-justify-xs-center MuiGrid-grid-xs-12"
      >
        <div
          class="MuiGrid-root MuiGrid-item MuiGrid-align-items-xs-flex-start"
        >
          <button
            class="MuiButtonBase-root MuiButton-root MuiButton-text"
            tabindex="0"
            type="button"
          >
            <span
              class="MuiButton-label"
            >
              <svg
                aria-hidden="true"
                class="MuiSvgIcon-root"
                focusable="false"
                viewBox="0 0 24 24"
              >
                <path
                  d="M11.67 3.87L9.9 2.1 0 12l9.9 9.9 1.77-1.77L3.54 12z"
                />
              </svg>
            </span>
            <span
              class="MuiTouchRipple-root"
            />
          </button>
        </div>
        <div
          class="MuiGrid-root makeStyles-title-2 MuiGrid-item"
        >
          <h1
            class="MuiTypography-root MuiTypography-h1"
          >
            Email Please
          </h1>
        </div>
      </div>
      <p>
        This is used to send you updates when a property is added and to reset your password if needed
      </p>
      <form>
        <div
          class="MuiFormControl-root MuiTextField-root MuiFormControl-fullWidth"
        >
          <label
            class="MuiFormLabel-root MuiInputLabel-root MuiInputLabel-formControl MuiInputLabel-animated"
            data-shrink="false"
            for="standard-basic"
            id="standard-basic-label"
          >
            Enter email
          </label>
          <div
            class="MuiInputBase-root MuiInput-root MuiInput-underline MuiInputBase-fullWidth MuiInput-fullWidth MuiInputBase-formControl MuiInput-formControl"
          >
            <input
              aria-invalid="false"
              class="MuiInputBase-input MuiInput-input"
              id="standard-basic"
              type="text"
              value=""
            />
          </div>
        </div>
        <div>
          <button
            aria-label="Submit Form"
            class="MuiButtonBase-root MuiIconButton-root"
            tabindex="0"
            type="button"
          >
            <span
              class="MuiIconButton-label"
            >
              <button
                class="MuiButtonBase-root MuiFab-root MuiFab-primary"
                tabindex="0"
                type="button"
              >
                <span
                  class="MuiFab-label"
                >
                  <svg
                    aria-hidden="true"
                    class="MuiSvgIcon-root"
                    focusable="false"
                    viewBox="0 0 24 24"
                  >
                    <path
                      d="M12 4l-1.41 1.41L16.17 11H4v2h12.17l-5.58 5.59L12 20l8-8z"
                    />
                  </svg>
                </span>
                <span
                  class="MuiTouchRipple-root"
                />
              </button>
            </span>
            <span
              class="MuiTouchRipple-root"
            />
          </button>
        </div>
      </form>
    </div>
  </body>,
  "container": <div>
    <div
      class="MuiGrid-root MuiGrid-container MuiGrid-justify-xs-center MuiGrid-grid-xs-12"
    >
      <div
        class="MuiGrid-root MuiGrid-item MuiGrid-align-items-xs-flex-start"
      >
        <button
          class="MuiButtonBase-root MuiButton-root MuiButton-text"
          tabindex="0"
          type="button"
        >
          <span
            class="MuiButton-label"
          >
            <svg
              aria-hidden="true"
              class="MuiSvgIcon-root"
              focusable="false"
              viewBox="0 0 24 24"
            >
              <path
                d="M11.67 3.87L9.9 2.1 0 12l9.9 9.9 1.77-1.77L3.54 12z"
              />
            </svg>
          </span>
          <span
            class="MuiTouchRipple-root"
          />
        </button>
      </div>
      <div
        class="MuiGrid-root makeStyles-title-2 MuiGrid-item"
      >
        <h1
          class="MuiTypography-root MuiTypography-h1"
        >
          Email Please
        </h1>
      </div>
    </div>
    <p>
      This is used to send you updates when a property is added and to reset your password if needed
    </p>
    <form>
      <div
        class="MuiFormControl-root MuiTextField-root MuiFormControl-fullWidth"
      >
        <label
          class="MuiFormLabel-root MuiInputLabel-root MuiInputLabel-formControl MuiInputLabel-animated"
          data-shrink="false"
          for="standard-basic"
          id="standard-basic-label"
        >
          Enter email
        </label>
        <div
          class="MuiInputBase-root MuiInput-root MuiInput-underline MuiInputBase-fullWidth MuiInput-fullWidth MuiInputBase-formControl MuiInput-formControl"
        >
          <input
            aria-invalid="false"
            class="MuiInputBase-input MuiInput-input"
            id="standard-basic"
            type="text"
            value=""
          />
        </div>
      </div>
      <div>
        <button
          aria-label="Submit Form"
          class="MuiButtonBase-root MuiIconButton-root"
          tabindex="0"
          type="button"
        >
          <span
            class="MuiIconButton-label"
          >
            <button
              class="MuiButtonBase-root MuiFab-root MuiFab-primary"
              tabindex="0"
              type="button"
            >
              <span
                class="MuiFab-label"
              >
                <svg
                  aria-hidden="true"
                  class="MuiSvgIcon-root"
                  focusable="false"
                  viewBox="0 0 24 24"
                >
                  <path
                    d="M12 4l-1.41 1.41L16.17 11H4v2h12.17l-5.58 5.59L12 20l8-8z"
                  />
                </svg>
              </span>
              <span
                class="MuiTouchRipple-root"
              />
            </button>
          </span>
          <span
            class="MuiTouchRipple-root"
          />
        </button>
      </div>
    </form>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
